frontends/web/src/components/devices/bitbox02/bitbox02.tsx

```diff
@@ -238,89 +239,91 @@ class BitBox02 extends Component<Props, State> {
+                                    title={t('bitbox02Wizard.initialize.title')}>
                                     {sdCardInserted ? true : this.insertSDCard()}
                                     <div className={style.standOut}>
                                         <img src={infoIcon} />
-                                        <span className={style.info}>Before continuing, it is highly recommended that you proceed in a secure environment.</span>
+                                        <span className={style.info}>{t('bitbox02Wizard.initialize.tip')}</span>
                                     </div>
                                     <div className={style.stepContext}>
-                                        <p>Successfully paired your BitBox! Now let's initialize your device. Get started by choosing to create a new wallet, or to restore a wallet from an existing backup. <strong>Please make sure you have a microSD card inserted in your BitBox.</strong></p>
+                                        <SimpleMarkup tagName="p" markup={t('bitbox02Wizard.initialize.text')} />
                                     </div>
                                     <div className={style.buttons}>
                                         <button
                                             className={[style.button, style.primary].join(' ')}
                                             onClick={this.createWalletStep}
                                             disabled={settingPassword || !sdCardInserted}>
-                                            Create Wallet
+                                            {t('bitbox02Wizard.initialize.createButton')}
                                     </button>
                                         <button
                                             className={[style.button, style.secondary].join(' ')}
                                             onClick={this.restoreBackupStep}
                                             disabled={!sdCardInserted}>
-                                            Restore Backup
+                                            {t('backup.restore.confirmTitle')}
                                     </button>
                                     </div>
                                 </Step> : ''}
                             {!unlockOnly && appStatus === 'createWallet' ?
                                 <Step
                                     active={createWalletStatus === 'intro'}
-                                    title="Create Wallet">
+                                    title={t('bitbox02Wizard.create.title')}>
                                     <div className={style.stepContext}>
-                                        <p>Ok, let's create a new wallet! Here are the basics steps you will be taking to setup your BitBox:</p>
+                                        <p>{t('bitbox02Wizard.create.text')}</p>
```

Comment:
Split out `Here are the basics steps you will be taking to setup your BitBox` into it's own field, as it's used twice (no need to have it be translated twice).